.helmignore

```diff
@@ -24,3 +24,6 @@
 *.tmproj
 
 .circleci/
+
+.doc/
```

Comment:
it doesn't trigger the right folder. It should be just `doc/` this cause the error `Error: INSTALLATION FAILED: create: failed to create: Request entity too large: limit is 3145728`. thanks

Reply:
Thank you for your contribution (again)